src/nft/components/sell/select/SelectPage.tsx

```diff
@@ -115,47 +150,169 @@ export const SelectPage = () => {
+        },
+        icon: <NonRarityIconFilled width="28" height="28" color={vars.color.blue400} />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+      {
+        displayText: 'Last price',
+        onClick: () => {
+          setOrderBy(false)
+          setSortBy(SortBy.LastPrice)
+        },
+        icon: <ClockIconFilled width="28" height="28" />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+      {
+        displayText: 'Date acquired',
+        onClick: () => {
+          setOrderBy(false)
+          setSortBy(SortBy.DateAcquired)
+        },
+        icon: <BagFillIcon width="28" height="28" color={vars.color.blue400} />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+      {
+        displayText: 'Date created',
+        onClick: () => {
+          setOrderBy(false)
+          setSortBy(SortBy.DateCreated)
+        },
+        icon: <PaintPaletteIconFilled width="28" height="28" color={vars.color.blue400} />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+      {
+        displayText: 'Date listed',
+        onClick: () => {
+          setOrderBy(false)
+          setSortBy(SortBy.DateListed)
+        },
+        icon: <TagFillIcon width="28" height="28" color={vars.color.blue400} />,
+        reverseOnClick: () => setOrderBy(!orderByASC),
+      },
+    ],
+    [orderByASC]
+  )
+
+  const SortWalletAssetsDropdown = () => <SortDropdown dropDownOptions={sortDropDownOptions} />
+
   return (
-    // Column style is temporary while we move over the filters bar that adjust width
-    <Column style={{ width: 'calc(100vw - 32px)' }}>
+    <Column width="full">
       <Row
         alignItems="flex-start"
         position="relative"
-        paddingLeft={{ sm: '0', md: '52' }}
+        paddingLeft={{ sm: '16', md: '52' }}
         paddingRight={{ sm: '0', md: '72' }}
         paddingTop={{ sm: '16', md: '40' }}
       >
-        <AnimatedBox paddingX="16" flexShrink="0" width="full">
-          <Row gap="8" flexWrap="nowrap">
-            <CollectionSearch searchText={searchText} setSearchText={setSearchText} />
-            <SelectAllButton />
-          </Row>
-          <InfiniteScroll
-            next={fetchNextPage}
-            hasMore={hasNextPage ?? false}
-            loader={
-              hasNextPage ? (
-                <Center>
-                  <LoadingSparkle />
-                </Center>
-              ) : null
-            }
-            dataLength={displayAssets.length}
-            style={{ overflow: 'unset' }}
+        <FilterSidebar SortDropdown={SortWalletAssetsDropdown} />
+
+        {(!isMobile || !isFiltersExpanded) && (
+          // @ts-ignore
+          <AnimatedBox
+            paddingLeft={isFiltersExpanded ? '24' : '16'}
+            flexShrink="0"
+            style={{
+              transform: gridX.interpolate(
+                (x) => `translate(${parseInt(`${x}`) - (!isMobile && isFiltersExpanded ? 300 : 0)}px)`
```

Comment:
you could also do as number instead of `parseInt` thats how they do it in other parts of the codebase

Reply:
also we should def make the filters size constant since its being reused

Reply:
updated to use Number and will work with design to standardize Filter width and update subsequently